doc/source/visualizing/callbacks.rst

```diff
@@ -622,8 +622,8 @@ Add the Current Time and/or Redshift
 
 .. function:: annotate_timestamp(x_pos=None, y_pos=None, corner='lower_left',\
                                  time=True, redshift=False, \
-                                 time_format='t = {time:.0f} {units}', \
-                                 time_unit=None, \
+                                 time_format='t = {time:.1f} {units}', \
```

Comment:
Why change to .1f?

Reply:
In the current codebase it actually is `.1f`, so now the docs match the code.

Reply:
Touché!  OK, sounds good!